fuzzer/retest-output

```diff
@@ -25,7 +25,7 @@ retestround() {
   grep "$JOINPATTERN" "$OUTFILE2" >> "$OUTFILE" && :
   grep "$TIMEOUTPATTERN" "$OUTFILE2" >> "$OUTFILE" && :
   cargo rustc --release -- -C target-cpu=native > /dev/null 2>&1
-  grep "$FINDPATTERN" < "$OUTFILE2" | sed 's/pattern: "\(.*\)"/\1/' | RUST_BACKTRACE=1 ./target/release/fuzzer --retest >> "$OUTFILE" 2>&1
+  grep "$FINDPATTERN" < "$OUTFILE2" | sed 's/pattern: \({.*}\)/\1/' | RUST_BACKTRACE=1 ./target/release/fuzzer --retest >> "$OUTFILE" 2>&1
```

Comment:
Either `pattern: \(Pattern {.*}\)` or `pattern: \(.*\)`, but the above doesn't match anything.

Reply:
Fixed!

Reply:
Wait, that doesn't make any sense… It should work now with the serde serialization. Let me test a bit more, as things seem weird…

Reply:
Nevermind, the problem was that my `output` file still had old patterns. Now it works as expected.

Reply:
Can happen, no worries!